portal-kernel/test/unit/com/liferay/portal/kernel/test/rule/NewEnvJVMTestRuleTest.java

```diff
@@ -228,7 +229,8 @@ private static String _toString(Map<String, String> map) {
 		for (Map.Entry<String, String> entry : map.entrySet()) {
 			sb.append(entry.getKey());
 			sb.append(_SEPARATOR_KEY_VALUE);
-			sb.append(entry.getValue());
```

Comment:
@julschong it should just be a if check here, on seeing value is null, don't append for value and the separator. Then on the receiving jvm, it will see the parts.length is 1, which automatically set a null value.

Reply:
@shuyangzhou after further investigation:
System.getenv() has this description: 
```
Returns an unmodifiable string map view of the current system environment. The environment is a system-dependent mapping from names to values which is passed from parent to child processes.
If the system does not support environment variables, an empty map is returned.
The returned map will never contain null keys or values
```
If there is not null value then I think we can just not deal with null value.
Blank value should just be copied as Blank still

Reply:
I will make the change

Reply:
resubbmitted here: liferay-core-infra#6916

Reply:
see https://github.com/brianchandotcom/liferay-portal/pull/149644